src/solvers/refinement/string_constraint_generator_transformation.cpp

```diff
@@ -36,22 +36,29 @@ string_exprt string_constraint_generatort::add_axioms_for_set_length(
 
   // We add axioms:
   // a1 : |res|=k
-  // a2 : forall i<k. (i<k ==> s[i]=s1[i]) &&(i >= k ==> s[i]=0)
+  // a2 : forall i<|s1|. i < k  ==> res[i] = s1[i]
+  // a3 : forall i<|s1|. i >= k ==> res[i] = 0
 
```

Comment:
This makes sense. I will replace `k` with `|res|` in the documentation of axioms `a2` and `a3`.